sdk/cognitivelanguage/ai-language-text/swagger/README.md

```diff
@@ -360,8 +360,7 @@ directive:
       $.description = "Options for a dynamic classification action.";
   - where-model: AbstractiveSummarizationTaskParametersBase
     transform:
-      $.properties.sentenceCount.description = "The max number of sentences to be part of the summary.";
-      $.properties.sentenceCount["x-ms-client-name"] = "maxSentenceCount";
+      $.properties.sentenceCount.description = "The approximate number of sentences to be part of the summary.";
```

Comment:
Actually we just need to remove the swagger transform all together and use the swagger documentation for the property.